shared_model/packages/javascript/example/admin@test.priv

```diff
@@ -0,0 +1 @@
+1d7e0a32ee0affeb4d22acd73c2c6fb6bd58e266c8c2ce4fa0ffe3dd6a253ffb
```

Comment:
It looks like these keys and index.js below are copy-pasted. Is it possible to reuse their versions from the root example directory?

Reply:
No, content in this folder distributed apart from Iroha code.